src/services/Teambuilding.ts

```diff
@@ -166,11 +167,11 @@ function findTeamAssignmentByLearnerId(
  */
 export async function processLabsApplication(
   labsApplicationSubmission: ILabsApplicationSubmission
-): Promise<void> {
+): Promise<IFinalLabsProject> {
   const oktaId = labsApplicationSubmission.oktaId;
   const labsApplication = labsApplicationSubmission;
```

Comment:
This is not a show stopper, but do we just want to rename the function argument rather than introduce a second (duplicate or same since it's a reference to the original value) variable with a shorter name?